app/src/main/res/values/preference_keys.xml

```diff
@@ -79,7 +79,7 @@
     <string name="preference_key_reading_lists_deleted_ids">readingListsDeletedIds</string>
     <string name="preference_key_reading_list_pages_deleted_ids">readingListPagesDeletedIds</string>
     <string name="preference_key_show_reading_lists_sync_prompt">showReadingListsSyncPrompt</string>
-    <string name="preference_key_reading_lists_first_time_sync">readingListsFirstTimeSync</string>
+    <string name="preference_key_reading_lists_first_time_sync">readingListsFirstTimeSyncV2</string>
```

Comment:
Clever-ish hack: Resetting the name of this "preference" will cause reading lists to be fully re-synced the next time any activity is launched (since BaseActivity checks and resets the value of this one-time parameter).